source/agora/consensus/state/ValidatorSet.d

```diff
@@ -55,7 +55,7 @@ public alias EnrollmentFinder = bool delegate (in Hash enroll_key, out Enrollmen
 
 /// A Height and PublicKey pair to represent expiring Validators
 public alias ExpiringValidator = Tuple!(Height, "enrolled_height",
-    PublicKey, "pubkey");
+    Hash, "utxo", PublicKey, "pubkey");
```

Comment:
Can you extract those changes (add8fd7a6a59f589abb60561dece48298e3e98bd / source/agora/consensus/state/ValidatorSet.d) in their own PR ? We can merge it immediately.

Reply:
Sure, I will.